apps/lidar_odometry_step_1/lidar_odometry.cpp

```diff
@@ -324,16 +324,16 @@ void lidar_odometry_gui()
                                        auto calibration = MLvxCalib::CombineIntoCalibration(idToSn, preloadedCalibration);
                                        auto data = load_point_cloud(fn.c_str(), true, params.filter_threshold_xy, calibration);
 
-                                       if (fn == laz_files.front())
-                                       {
-                                           fs::path calibrationValidtationFile = wdp / "calibrationValidation.asc";
+                                    //    if (fn == laz_files.front())
```

Comment:
Please do not remove this functionality. It is crutial for calibration validation. Try to consider saving e.g. 1000.000 points.